### ISSUE TITLE
docker-solr 7.2.1

### ISSUE THREAD

**Issue by morissm** [docker-solr/docker-solr#162]
Solr 7.2.1 was quietly released yesterday. I'm hoping to see an equivalent docker-solr release.

**Comment by makuk66** [comment]
Of course; the change for that already went 14h ago https://github.com/docker-solr/docker-solr/commit/34a3a5da6995425a1fee38e58cc32210f069f579

Got one other issue to figure out first though, so ETA perhaps middle/late in the week. If you can't wait, use dockersolr/docker-solr:7.2.1

**Comment by makuk66** [comment]
https://github.com/docker-library/official-images/pull/3914

**Comment by makuk66** [comment]
`solr:7.2.1` is now available

**Comment by morissm** [comment]
Thanks!
